portfolio/src/main/java/com/google/sps/data/CommentResult.java

```diff
@@ -14,22 +14,18 @@
 
 package com.google.sps.data;
 
-/** Class containing quotes and the person who said it. */
-public final class QuotePerson {
+import com.google.sps.data.Comment;
+import java.util.ArrayList;
+import java.util.List;
 
-  private final String quote;
-  private final String person;
+/** Class containing the author, content, and timestamp of a comment. */
+public final class CommentResult {
 
-  public QuotePerson(String quote, String person) {
-    this.quote = quote;
-    this.person = person;
-  }
-
-  public String getQuote() {
-    return quote;
-  }
+  private final List<Comment> comments;
+  private final int count;
```

Comment:
Could you make the variable name more specific? 
Is this just the size of List<Comment> comments?

Reply:
Good idea, I will make that more clear.